cpp/tests/community/mg_louvain_test.cpp

```diff
@@ -256,8 +257,10 @@ INSTANTIATE_TEST_SUITE_P(
   ::testing::Combine(
     // enable correctness checks for small graphs
     ::testing::Values(Louvain_Usecase{100, 1, true}),
-    ::testing::Values(cugraph::test::File_Usecase("test/datasets/karate.mtx"),
-                      cugraph::test::File_Usecase("test/datasets/dolphins.mtx"))));
+    ::testing::Values(cugraph::test::File_Usecase(
+      "test/datasets/karate.mtx")  //,
+                                   // cugraph::test::File_Usecase("test/datasets/dolphins.mtx")
+                      )));
```

Comment:
Is there any particular reason for not testing with dolphin.mtx?

Reply:
Nope.  I had commented it out to do faster debugging on something that only failed in karate.  I'll add that back in.